include/deal.II/grid/grid_tools.h

```diff
@@ -2632,7 +2632,36 @@ namespace GridTools
   copy_material_to_manifold_id(Triangulation<dim, spacedim> &tria,
                                const bool compute_face_ids = false);
 
-
+  /**
+   * Propagate manifold indicators associated to the cells of the Triangulation
+   * @p tria to their co-dimension one and two objects.
+   *
+   * This function sets the @p manifold_id of shared faces and edges to the value
+   * returnd by the @p disambiguation_function method, called with the set of
+   * manifold indicators of the cells that share the same face or edge.
+   *
+   * By default, the @p disambiguation_function returns
+   * numbers::flat_manifold_id when the set has size greater than one (i.e.,
```

Comment:
actually, it returns `invalid_manifold_id` :-)

Reply:
not any more. :)